qCC/mainwindow.cpp

```diff
@@ -8857,6 +8859,116 @@ void MainWindow::doActionCloudMeshDist()
 	freezeUI(true);
 }
 
+void MainWindow::doActionCloudPrimitiveDist()
+{
+	bool foundPrimitive = false;
+	unsigned primitiveIndx;
+	ccHObject::Container clouds;
+	clouds.clear();
+	ccPlane* refPlane = nullptr;
+	ccSphere* refSphere = nullptr;
+	QString PrimitiveName;
```

Comment:
the (implicit :D) coding rule in CC is to start variables by a lower case letter (upper case letters are for class names or static methods)

Reply:
Sorry about that, not my standard style so I slip into old habits thanks for the review.
BTW I am positive this won't be the last time this happens